db/schema.rb

```diff
@@ -12,7 +12,7 @@
 
 ActiveRecord::Schema.define(version: 2021_01_12_023923) do
 
-  create_table "active_storage_attachments", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+  create_table "active_storage_attachments", charset: "latin1", force: :cascade do |t|
```

Comment:
btw, running `ALTER DATABASE littlesis CHARACTER SET = utf8mb4 COLLATE = utf8mb4_unicode_ci;` will change the default charset, which will prevent these changes in schema.rb from happening in the future

Reply:
Oops. Yes, thanks. I had noted this happening myself. Bizarre that it is still defaulting to what is essentially ASCII.